src/lib/krb5/os/sendto_kdc.c

```diff
@@ -800,7 +805,7 @@ resolve_server(krb5_context context, const krb5_data *realm,
         ai.ai_addr = (struct sockaddr *)&entry->addr;
         defer = (entry->transport != transport);
         return add_connection(conns, entry->transport, defer, &ai, ind, realm,
-                              NULL, entry->uri_path, udpbufp);
+                              NULL, NULL, entry->uri_path, udpbufp);
```

Comment:
Is HTTPS transport supported for entry->hostname == NULL? I argue that the HTTPS transport requires a hostname for SNI and virtual hosting.